verifier_common/src/lib.rs

```diff
@@ -12,20 +12,25 @@ compile_error!("multiple security levels selected same time");
 pub const SECURITY_BITS: usize = 80;
 #[cfg(feature = "security_80")]
 pub const POW_BITS: usize = 28;
+#[cfg(feature = "security_80")]
+pub const MEMORY_DELEGATION_POW_BITS: usize = pow_bits_for_memory_and_delegation(
+    SECURITY_BITS,
+    cs::one_row_compiler::MAX_NUMBER_OF_CYCLES.leading_zeros() as usize,
+    MERSENNE31QUARTIC_SIZE_LOG2,
+);
 
 #[cfg(feature = "security_100")]
 pub const SECURITY_BITS: usize = 100;
 #[cfg(feature = "security_100")]
 pub const POW_BITS: usize = 28;
+#[cfg(feature = "security_100")]
+pub const MEMORY_DELEGATION_POW_BITS: usize = pow_bits_for_memory_and_delegation(
+    SECURITY_BITS,
+    cs::one_row_compiler::MAX_NUMBER_OF_CYCLES.leading_zeros() as usize,
```

Comment:
Does it build in no-std? If we import `cs` as "definitions only", then "one row compiler" module is not available? Also, leading zeroes are very misleading. If the number of cycles is not a power of two, instead it's better to take "next power of two", and then trailing zeroes